packages/editor-ui/src/plugins/telemetry/index.ts

```diff
@@ -74,7 +74,11 @@ export class Telemetry {
 	}
 
 	identify(instanceId: string, userId?: string, versionCli?: string) {
```

Comment:
might be good place to start adding unit tests

Reply:
added tests to the track method!